includes/Modules/Analytics_4.php

```diff
@@ -194,6 +195,14 @@ protected function get_datapoint_definitions() {
 	 * @throws Invalid_Datapoint_Exception Thrown if the datapoint does not exist.
 	 */
 	protected function create_data_request( Data_Request $data ) {
+		$prefix_account_id = function( $account_id ) {
+			return 'accounts/' . $account_id;
+		};
+
+		$prefix_property_id = function( $property_id ) {
+			return 'properties/' . $property_id;
+		};
```

Comment:
Can we make these two private or protected methods? While this is currently only used in this one method, it's generic functionality for the module, and we should work with methods/functions rather than closures unless they are super-contextual, which e.g. also requires proper method docs.

Reply:
I have made them public static methods.